FluentTerminal.App/Views/SettingsPages/ShellProfileSettings.xaml

```diff
@@ -264,6 +264,19 @@
                                     <HyperlinkButton Command="{x:Bind AddKeyboardShortcutCommand}" Content="Add" />
                                 </StackPanel>
                                 <views:KeyBindingsView ShowCommandName="False" ViewModel="{x:Bind KeyBindings}" />
+                                <TextBlock Margin="0,0,0,8" Text="Line ending translation on paste" />
```

Comment:
Since the "Keyboard shortcut" setting can grow in size it feels awkward to me that this setting is placed below that. Personally I would move this setting above the "Keyboard shortcut" setting.

If you update this, don't forget to update it [here](https://github.com/felixse/FluentTerminal/blob/4349d3ad44169f4de38365a3fc00568f47edbeaa/FluentTerminal.App/Views/SettingsPages/ShellProfileSettings.xaml#L324) as well.